build/Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN \
     # Note: procps for "top" command (useful when debugging processes).
     # Note: tini is a helper for reaping zombie processes.
     apt-get update -qq &&\
-    DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends curl gnupg procps tini python3 default-jre-headless &&\
+    DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends curl gnupg procps tini python3 qpdf default-jre-headless &&\
```

Comment:
Could you move the installation instructions on a dedicated line? 😄 
For instance, after PDFtk installation?

Reply:
Solved